flair/embeddings.py

```diff
@@ -488,6 +493,57 @@ def __str__(self):
         return self.name
 
 
+class TransformerXLEmbeddings(TokenEmbeddings):
+    def __init__(self,
+                 model: str = 'transfo-xl-wt103'):
+        """Transformer-XL embeddings, as proposed in Dai et al., 2019.
+        :param model: name of Transformer-XL model
+        """
+        super().__init__()
+
+        if model not in TRANSFORMER_XL_PRETRAINED_MODEL_ARCHIVE_MAP.keys():
+            raise ValueError('Provided Transformer-XL model is not available.')
+
+        self.tokenizer = TransfoXLTokenizer.from_pretrained(model)
+        self.model = TransfoXLModel.from_pretrained(model)
+        self.name = model
+        self.static_embeddings = True
+
+        dummy_sentence: Sentence = Sentence()
+        dummy_sentence.add_token(Token('hello'))
+        embedded_dummy = self.embed(dummy_sentence)
+        self.__embedding_length: int = len(embedded_dummy[0].get_token(1).get_embedding())
+
+    @property
+    def embedding_length(self) -> int:
+        return self.__embedding_length
+
+    def _add_embeddings_internal(self, sentences: List[Sentence]) -> List[Sentence]:
+        self.model.to(flair.device)
+        self.model.eval()
+
+        with torch.no_grad():
+            for sentence in sentences:
+                indexed_tokens = self.tokenizer.convert_tokens_to_ids(sentence.tokens)
```

Comment:
This line causes an error since `sentence.tokens` returns a list of token representation strings with meta information. So instead of `['I', 'love', 'Berlin']` it tries to embed  `[Token: 1 I, Token: 2 love, Token: 3 Berlin]` which produces weird embeddings.


Replace with: 

```python
                token_strings = [token.text for token in sentence.tokens]
                indexed_tokens = self.tokenizer.convert_tokens_to_ids(token_strings)

```

Reply:
I see, sorry for that!